src/main/resources/application.properties

```diff
@@ -46,6 +46,8 @@ com.ta.reportportal.job.interrupt.broken.launches.cron=PT1H
 #CRON expression for project setting 'keep screenshots' as every 7 days
 #com.ta.reportportal.job.clean.screenshots.cron=P7D
 com.ta.reportportal.job.clean.screenshots.cron=PT1H
+com.ta.reportportal.job.load.plugins.cron=PT1M
```

Comment:
it just for debug isn't it?

Reply:
i mean 1 minute

Reply:
This job should be executed frequently, because we will need to distribute between all replicas plugin that was uploaded to some replica.
And it's not high-loaded component, because if there are no plugins to upload this piece of code will just execute read operation with few results